code/game/machinery/computer/prisoner/management.dm

```diff
@@ -26,7 +26,7 @@
 		if(contained_id)
 			dat += text("<A href='?src=[REF(src)];id=eject'>[contained_id]</A><br>")
 			dat += text("Collected Points: [contained_id.points]. <A href='?src=[REF(src)];id=reset'>Reset.</A><br>")
-			dat += text("Card goal: [contained_id.goal].  <A href='?src=[REF(src)];id=setgoal'>Set </A><br>")
+			dat += text("Card goal: [contained_id.goal].  <A href='?src=[REF(src)];id=setgoal'>Set</A> <A href='?src=[REF(src)];id=setpermanent'>Make permanent</A><br>")
```

Comment:
Shouldn't `Make Permanent` instead be a ternary that checks if it's already permanent or not and has different text if it's already permanent?